Runner/CustomTemplateFactory.py

```diff
@@ -0,0 +1,170 @@
+import json
```

Comment:
📓 Please note if this is needed in your project: `# -*- coding: utf-8 -*-`

* https://docs.python.org/3/reference/lexical_analysis.html#encoding-declarations

Reply:
will do